### PR TITLE
switch to flint and bump version to 1.0

### PR DESCRIPTION
**Pull request by gertjanvanzwieten**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "bottombar"
+authors = [{name = "Evalf", email = "info@evalf.com"}]
+readme = "README.md"
+license = {file = "LICENSE"}
+classifiers = ["License :: OSI Approved :: MIT License"]
+dynamic = ["version", "description"]
```
Comment (joostvanzwieten) [inline pyproject.toml:12]: Using a dynamic description, flit takes the first line of the module docstring, which currently is 'BottomBar is a context manager that prints a status line at the bottom of a' (to reproduce: build a wheel and open `bottombar-1.0.dist-info/METADATA`, line starting with `Summary:`). I suggest to start the docstring with a single line, e.g. 'Context manager that prints a status line at the bottom of a terminal`, then a blank line and the current contents of the docstring.

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
```
Comment (joostvanzwieten) [inline pyproject.toml:13]: Maybe we should add `requires-python = ">=3"` as well. Currently Flit builds a wheel that advertises support for Python 2 and 3.

File: pyproject.toml (hunk 1 above)
Comment (gertjanvanzwieten) [reply pyproject.toml:12]: Changed module doc to "Print and maintain a status line at the bottom of a VT100 terminal." and moved remainder to the BottomBar class.

File: pyproject.toml (hunk 2 above)
Comment (gertjanvanzwieten) [reply pyproject.toml:13]: done

**Review by joostvanzwieten — APPROVED** [review]

**Pull request merged**